packages/api/src/services/stats/lsp.ts

```diff
@@ -73,6 +73,19 @@ export default (config: Config, appState: Dependencies) => {
       })
     );
   }
+  // TVM for LSP will just be the TVL
+  async function updateTvm(address: string) {
+    const tvl = await stats[currency].latest.tvl.get(address);
+    return stats[currency].latest.tvm.upsert(address, tvl);
```

Comment:
this stats/lsp file is where the tvm values get updated and stored. This particular function just moves the tvl to tvm and nothing more. If our TVM calculation ever changes, this is the only function which would need to update how we calculate that value